cvat/apps/engine/serializers.py

```diff
@@ -1002,6 +1002,19 @@ class LogEventSerializer(serializers.Serializer):
 class AnnotationFileSerializer(serializers.Serializer):
     annotation_file = serializers.FileField()
 
+class TusUploadingMetaSerializer(serializers.Serializer):
+    """
+    Allows to specify TUS file uploading details.
+
+    To state that the input files are sent in the correct order,
+    pass an empty list of files in the 'files' field. A list of
+    strings is expected to be a list file names in the required order.
+    """
+
+    files = serializers.ListField(
+        child=serializers.CharField(max_length=1024), default=[], allow_empty=True)
```

Comment:
```suggestion
        child=serializers.CharField(max_length=1024), default=list, allow_empty=True)
```

To avoid sharing the same list between multiple instances.